pkg/runtime/depot.go

```diff
@@ -28,9 +28,10 @@ type depotConfig struct {
 }
 
 type deployment struct {
-	Type  deploymentType `json:"type"`
-	Path  string         `json:"path"`
-	Files []string       `json:"files"`
+	Type       deploymentType `json:"type"`
+	Path       string         `json:"path"`
+	Files      []string       `json:"files"`
+	InstallDir string         `json:"installDir"`
```

Comment:
This is needed to compute the on-disk path to a depot artifact deployment's relative sources. envdef usually provides a root directory like "installdir", and we need to retain that value to compute source paths outside of setup (i.e. when trying to re-link or re-copy files from another artifact).